snuba/datasets/configuration/issues/storages/search_issues.yaml

```diff
@@ -114,6 +114,12 @@ query_processors:
         contexts:
           trace.trace_id: trace_id
 
+deletion_settings:
+  is_enabled: 0
```

Comment:
should is_enabled = 1 for true?

Reply:
I was going to leave this as `0` until we were ready to test it out (and until we had the killswitch implemented)